README.md

```diff
@@ -14,3 +14,4 @@ Available packages:
 Package|Description
 ---|---
 cf-cli|The official Cloud Foundry CLI
+bosh-init|The official CLI to create and updates the Bosh Director VM
```

Comment:
* Please fix typo:
The official CLI to create and update~~**s**~~ the Bosh Director VM
* Please consider using uppercase `BOSH` to be consistent with usage on bosh.io?